src/program/lwaftr/migrate_configuration/migrate_configuration.lua

```diff
@@ -483,7 +483,7 @@ local function multiprocess_migration(src, conf_file)
    local instance = {
       [device] = {queue = cltable.new({ key_type = queue_key }),},
    }
-   local key = ffi.new(queue_key, 1)
+   local key = ffi.new(queue_key, 0)
```

Comment:
Just a question on this. Do you think it's sufficient to update this migration or do you think it requires a new migration converting the 1-id based list to a 0-id based list?

Reply:
I think it's sufficient to update the migration.  Apparently it's perfectly fine for a device to be configured with one queue with id=1 without an id=0.  Configurations migrated as id=1 will continue to work just fine.  I updated the .conf files in the repo just so they would match what a migration would have produced, rather than out of any real need!

Reply:
Okay cool, sounds good :)